x-pack/plugins/elastic_assistant/server/routes/knowledge_base/entries/bulk_actions_route.ts

```diff
@@ -201,8 +201,14 @@ export const bulkActionKnowledgeBaseEntriesRoute = (router: ElasticAssistantPlug
             docs_deleted: docsDeleted,
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
           } = await writer!.bulk({
-            documentsToCreate: body.create?.map((c) =>
-              transformToCreateSchema(changedAt, spaceId, authenticatedUser, c)
+            documentsToCreate: body.create?.map((entry) =>
+              transformToCreateSchema({
+                createdAt: changedAt,
+                spaceId,
+                user: authenticatedUser,
+                entry,
+                // TODO: Add V2 flag
```

Comment:
No longer needed as there are legacy transform functions that can be used if needed (not here though since this API is behind the FF). Will remove.